shuffle/cli/src/account.rs

```diff
@@ -80,6 +140,24 @@ pub fn handle(project_dir: PathBuf, account_key_path: PathBuf) -> Result<()> {
         ::hex::encode(new_account.private_key().to_bytes())
     );
     println!("Public key: {}", new_account.public_key());
+    Ok(())
+}
 
+#[derive(Debug, StructOpt)]
+pub enum AccountCommand {
+    // Rotates dev.key with new private key and saves old key in /keys
+    #[structopt(
+        about = "Rotates private key and creates new account with new key. Use rotate if you don't have a dev.key"
+    )]
+    Rotate,
+    //     TODO: Add List subcommand
```

Comment:
Let's remove this on this PR

Reply:
Are you referring to removing the whole subcommand and reverting back to the -r flag that I had set up before?